src/api/main.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import FastAPI
+
+from .routers import categories, transactions
+
+description = """
+This API supports the Finance Tracker application frontend.
+Data for the application can be manipulated directly through this API.
+
+## Categories 🗃️
+
+You can **create**, **read**, **update**, and **delete** categories. Categories are used to classify transactions.
+
+## Transactions 💸
+
+You can **create**, **read**, **update**, and **delete** transactions.
+
+## Reports 📈
+
+You can **generate** reports to analyze your financial data. Reports can be filtered by date range and category.
+
+[ IN PROGRESS ]
+"""
+
+tags_metadata = [
+    {
+        "name": "categories",
+        "description": "Manage categories.",
+    },
+    {
+        "name": "transactions",
+        "description": "Manage transactions.",
+    },
+]
+
+app = FastAPI(
+    title="Finance Tracker 💰",
+    description=description,
+    summary="Record transactions and audit your finances against a monthly budget.",
+    version="0.0.1",
+    contact={
+        "name": "Kishan Patel",
+        "url": "https://kishanpatel.dev",
+        "email": "kishanpatel789@gmail.com",
```

Comment:
wonder if this meta data could be loaded from pyproject.toml to have one source of truth? not sure what's best practice?